pkgs/pubspec_parse/lib/src/dependency.dart

```diff
@@ -279,3 +267,18 @@ class HostedDetails {
 
 VersionConstraint _constraintFromString(String? input) =>
     input == null ? VersionConstraint.any : VersionConstraint.parse(input);
+
+/// The `value` if it is a `Map`, or `{'uri': value}` if it is a `String`.
```

Comment:
Doc and code don't agree...


```suggestion
/// The `value` if it is a `Map`, or `{'url': value}` if it is a `String`.
```